### ISSUE TITLE
Add scylla.yaml configuration to scylla_setup

### ISSUE THREAD

**Issue by tzach** [scylladb/scylla#3181]
*Installation details*
Scylla version (or git commit hash): 2.1

Scylla setup consists of two parts (for example for [CentOS](http://www.scylladb.com/download/centos_rpm/))
1. running scylla_setup, for OS and Scylla configuration
2. updating scylla.yaml parameters:
 * cluster_name
 * seeds
 * listen_address
 * rpc_address

It will be easier for the user to include the setting of these parameters in scylla_setup itself:
- in the interactive version, the prompt can suggest listen_address address, for example, "localhost."
- the command line, non-interactive option, will make it easier to automate teh installation without a need for yaml manipulation.

**Comment by dorlaor** [comment]
+1 !

**Comment by Faraknight** [comment]
+1

**Comment by gnumoreno** [comment]
+1

**Comment by yaronkaikov** [comment]
@tzach Is this still relevant ?

**Comment by tzach** [comment]
Yes! I forgot about this, but its look like Tzach from 6 years ago made a valid suggestion.
